FundacionParaguaya/MentorApp
# Feature/demo survey no mandatory

**Describe the changes**
Added validation schema based on `requiredFields` in `surveyConfig`
Delete draft on exit if is in Demo list
Disable submitting if is Demo
Skip SocioEconomic screen for UK Short demo

 
**To Test**
Demo Surveys : UK - SHORT DEMO, Stoplight Demo - Spanish ,Stoplight Demo - English -- take all of these surveys and try exiting the draft should be deleted on exit. When you get to the Final screen you should not see "Close" button. In UK SHORT DEMO survey you should not see Economic questions screen. In Spanish / English Demo you fields for Primary Participant and Family Members should not be required. Try taking different survey than those 3. Exit draft and see if it saves. Try submitting at the Final screen


This issue also solves #880 and #891
